lib/apiservers/engine/proxy/archive.go

```diff
@@ -157,23 +156,29 @@ func (a *ArchiveProxy) ArchiveImportWriter(ctx context.Context, store, deviceID
 			params = params.WithFilterSpec(&encodedFilter)
 		}
 
+		var plErr error
 		_, err = a.client.Storage.ImportArchive(params)
 		if err != nil {
 			switch err := err.(type) {
 			case *storage.ImportArchiveInternalServerError:
-				plErr := InternalServerError(fmt.Sprintf("Server error from archive writer for device %s", deviceID))
-				log.Errorf(plErr.Error())
+				plErr = InternalServerError(fmt.Sprintf("Server error from archive writer for device %s", deviceID))
```

Comment:
This may be a translation of the current error, but I don't think it's useful to the user. Can we change to something such as:
`error writing files to %s`